mediacloud/mediawords/util/web/user_agent/throttled.py

```diff
@@ -0,0 +1,67 @@
+"""Implement ThrottledUserAgent as a sub class of mediawords.util.web.UserAgent with per domain throttling."""
+
+import typing
+
+import mediawords.db
+import mediawords.util.config
+from mediawords.util.web.user_agent import UserAgent
+from mediawords.util.web.user_agent.request.request import Request
+from mediawords.util.web.user_agent.response.response import Response
+
+from mediawords.util.log import create_logger
+log = create_logger(__name__)
+
+# default amount of time in between requests
+_DEFAULT_DOMAIN_TIMEOUT = 10
```

Comment:
should this be in a global config?

Reply:
I find it clearer just to have all of the constants at the top of the file unless there's a reason to isloate them.